src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsProvider.java

```diff
@@ -146,7 +153,10 @@ public void deleteInstance(Infrastructure infrastructure, String instanceId) {
         try {
             execResponses = getComputeServiceFromInfastructure(infrastructure).runScriptOnNodesMatching(runningInGroup(instanceTag),
                                                                                                         buildScriptToExecuteString(instanceScript),
-                                                                                                        buildScriptOptions(instanceScript));
+                                                                                                        buildScriptOptions(instanceScript,
+                                                                                                                           null,
+                                                                                                                           infrastructure,
+                                                                                                                           instanceTag));
         } catch (Exception e) {
             throw new RuntimeException(e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)